src/GitInfo.macro.js

```
@@ -18,6 +18,11 @@ const gitInfo = (() => {
   } catch (e) {
     throw Error(`Unable to parse the git information: ${e}`);
   }
+  try {
+    // try and get an exact tag. this is expected to throw if there is no tag
+    ret.tag = execSync('git describe --tags --exact-match').toString().trim();
+  } catch(_) {}
```

Ignoring all errors may hide unexpected issues. Probably better to `console.warn` it out.

---

The command is now within the main try/catch block